src/app/components/core/configuration-manager/configuration-group/configuration-group.component.css

```diff
@@ -15,6 +15,24 @@
   overflow: visible;
 }
 
+.custom-quickview-configuration {
+  border: none !important;
+}
+
+.custom-quickview-card {
+  box-shadow: none;
+  margin-bottom: 24px;
+}
+
+.custom-quickview-header {
+  box-shadow: none;
+  background-color: white;
+}
+
+.custom-quickview-title {
+  padding-left: 0px !important;
+}
+
```

Comment:
this should probably go to a common place.

Reply:
These classes are for child component which is projected in quickview through ng-content. These will be different for each projected component.